src/Tasks/ProcessJobQueueTask.php

```diff
@@ -83,7 +85,7 @@ public function run($request)
         }
 
         // Run the queue
-        $queue = $this->getQueue($request);
+        $queue = AbstractQueuedJob::getQueue($request->getVar('queue') ?? 'Queued');
```

Comment:
```suggestion
        $queue = AbstractQueuedJob::getQueue($request->getVar('queue') ?? 'queued');
```

Reply:
`Queued` is taken from the original method I'm moving this from. I'm hesitant to change it as it's not clear if it was like that for a good reason. Though I guess it goes through `strtolower` so it doesn't matter whether it's caps or not. I'd prefer to just leave it.